cmd/dex/config.go

```diff
@@ -47,7 +47,8 @@ type Config struct {
 	// StaticPasswords cause the server use this list of passwords rather than
 	// querying the storage. Cannot be specified without enabling a passwords
 	// database.
-	StaticPasswords []password `json:"staticPasswords"`
+	StaticPasswords  []password `json:"staticPasswords"`
+	UserInfoEndpoint string     `json:"userInfoEndpoint"`
```

Comment:
Plz add a comment to describe the UserInfoEndpoint. A link to https://openid.net/specs/openid-connect-discovery-1_0.html#ProviderMetadata would also be helpful.